### DISCUSSION TITLE
Do you want OpenCalc to implement unit convertor in the future?

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#336] [Polls]
Binary - #240, Weight/Mass, Length, Volume, Speed, Pressure, Area, Time, Temperature, Angle, Data, Power

Plus Conversion Cheatsheet from #50 

Plus Bmi calculator, Date calculator, Time zones calculator, Roman numeral converter

**Comment by inson1** [comment]
Maybe there should be OpenCalc Lite (only with calculator) and OpenCalc (with convertors etc.) in the future

**Reply by albertgarcesb96** [reply]
I think it would be a better idea to develop an Unit Converter separately and suggest the user to install it as an option and at the same time once it is installed it works as an add-on and its icon can be hidden from the application drawer.

**Comment by inson1** [comment]
This [screenshot](https://d4.alternativeto.net/5nMnpYFL-8J9VvWYCUIpMXR8VuPhBsjgPB18nH9McQY/rs:fit:133:288:0/g:ce:0:0/YWJzOi8vZGlzdC9zL2NvbS1jYXB5YmFyYS1jYWxjdWxhdG9yXzYwNjc1N19mdWxsLmpwZw.jpg) looks nice for inspiration.
https://alternativeto.net/software/com-capybara-calculator/about/
https://github.com/rajput-hemant/calculator

**Comment by inson1** [comment]
And maybe graph support
